aten/src/ATen/native/cudnn/LossCTC.cpp

```diff
@@ -62,7 +62,13 @@ std::tuple<Tensor, Tensor> _cudnn_ctc_loss(const Tensor& log_probs_t, const Tens
 
   cudnnCTCLossAlgo_t algo = (deterministic ? CUDNN_CTC_LOSS_ALGO_DETERMINISTIC : CUDNN_CTC_LOSS_ALGO_NON_DETERMINISTIC);
 
-  Tensor probs = log_probs->softmax(2);
+  Tensor probs;
+  // before 7.6 passing probs was the only option, starting with 7.1 the log probs/unnormalized log probs is the default
```

Comment:
starting with 7.6? Nice find, it's not in the release notes unfortunately

Reply:
Unfortunately, the fix isn't working, either, as one would need to set the non-default option or somesuch.